src/ORM/Marshaller.php

```diff
@@ -105,6 +105,22 @@ public function one(array $data, array $options = [])
         $propertyMap = $this->_buildPropertyMap($options);
 
         $schema = $this->_table->schema();
+        $primaryKey = $schema->primaryKey();
+
+        if (array_intersect($primaryKey, array_keys($data)) == $primaryKey) {
+            $tableName = $this->_table->alias();
+            $record = $this->_table->find('all');
+            foreach ($primaryKey as $pkey) {
+                $record->where(["$tableName.$pkey" => $data[$pkey]]);
```

Comment:
Prefixing field with table name/alias similar to `"$tableName.$pkey"` will be often needed by users. It be nice to have `Table::escapeField()` similar to `Model::escapeField()` in 2.x.